README.md

```diff
@@ -117,11 +117,12 @@ dirctl build [options] <path-to-agent>
 Options:
 - `--name` : Name of the agent.
 - `--version` : Version of the agent.
-- `--artifact` : Artifacts to set for the agent. Each artifact should be in the format `type:url`. Example usage: `--artifact type1:url1 --artifact type2:url2`. Supported types: `docker-image`, `python-package`, `helm-chart`, `source-code` and `binary`.
+- `--locator` : Artifact locators to set for the agent. Each locator should be in the format `type:url`. Example usage: `--locator type1:url1 --locator type2:url2`. Supported types: `docker-image`, `python-package`, `helm-chart`, `source-code` and `binary`.
```

Comment:
lets maybe keep the --artifact notation until next week. we can resolve the failing CSIT tests once we release a v0.1.4 without causing a breaking change again